src/gui/src/timingWidget.cpp

```diff
@@ -0,0 +1,394 @@
+  path_widget->addTab(bottom_widget, "Path Details");
+  QGridLayout* bottom_widg_layout = new QGridLayout;
+  bottom_widget->setLayout(bottom_widg_layout);
+  QFrame* frame = new QFrame;
+  frame->setFrameShape(QFrame::StyledPanel);
+  frame->setFrameShadow(QFrame::Raised);
+  bottom_widg_layout->addWidget(frame);
+
+  QHBoxLayout* frame_layout = new QHBoxLayout;
+  frame->setLayout(frame_layout);
+  frame_layout->addWidget(new QLabel("Find"));
+  frame_layout->addWidget(find_object_edit_);
+  find_object_edit_->setFocusPolicy(Qt::ClickFocus);
+  find_object_edit_->setPlaceholderText("Pin or Net");
+
+  frame_layout->addWidget(new QLabel("Path:"));
+  frame_layout->addWidget(path_index_spin_box_);
+
+  bottom_widg_layout->addWidget(path_details_table_view_);
+
+  layout->addWidget(path_widget, 2, 0);
+
+  container->setLayout(layout);
+  setWidget(container);
+
+  connect(setup_timing_table_view_->horizontalHeader(),
+          SIGNAL(sectionClicked(int)),
+          this,
+          SLOT(timingPathsViewCustomSort(int)));
+  connect(hold_timing_table_view_->horizontalHeader(),
+          SIGNAL(sectionClicked(int)),
+          this,
+          SLOT(timingPathsViewCustomSort(int)));
+  connect(path_details_table_view_,
+          SIGNAL(clicked(const QModelIndex&)),
+          this,
+          SLOT(highlightPathStage(const QModelIndex&)));
+
+  connect(find_object_edit_,
+          SIGNAL(returnPressed()),
+          this,
+          SLOT(findNodeInPathDetails()));
+
+  connect(
+      path_index_spin_box_, SIGNAL(valueChanged(int)), this, SLOT(showPathIndex(int)));
+
+  connect(dbchange_listener_,
+          SIGNAL(dbUpdated(QString, std::vector<odb::dbObject*>)),
+          this,
+          SLOT(handleDbChange(QString, std::vector<odb::dbObject*>)));
+  connect(
+      update_button_, SIGNAL(clicked()), this, SLOT(populatePaths()));
+
+  connect(
+      setup_timing_table_view_->selectionModel(),
+      SIGNAL(selectionChanged(const QItemSelection&, const QItemSelection&)),
+      this,
+      SLOT(selectedRowChanged(const QItemSelection&, const QItemSelection&)));
+
+  connect(
+      hold_timing_table_view_->selectionModel(),
+      SIGNAL(selectionChanged(const QItemSelection&, const QItemSelection&)),
+      this,
+      SLOT(selectedRowChanged(const QItemSelection&, const QItemSelection&)));
+
+  connect(
+      path_details_table_view_->selectionModel(),
+      SIGNAL(selectionChanged(const QItemSelection&, const QItemSelection&)),
+      this,
+      SLOT(selectedDetailRowChanged(const QItemSelection&,
+                                    const QItemSelection&)));
+
+  connect(render_enable_button_, SIGNAL(toggled(bool)), this, SLOT(toggleRenderer()));
+
+  connect(setup_timing_paths_model_, SIGNAL(modelReset()), this, SLOT(modelWasReset()));
+  connect(hold_timing_paths_model_, SIGNAL(modelReset()), this, SLOT(modelWasReset()));
+
+  path_index_spin_box_->setRange(0, 0);
+}
+
+TimingWidget::~TimingWidget()
+{
+}
```

Comment:
Is this needed?

Reply:
probably not, that was there before, but also empty, I think it can be safely removed.